src/allmydata/client.py

```diff
@@ -38,7 +36,7 @@
 PiB=1024*TiB
 
 def _valid_config_sections():
-    cfg = _common_config_sections()
+    cfg = node._common_config_sections()
```

Comment:
It might just be me habitually putting "_" in front now so I don't have to support the API without thinking about it further ;)
(I've been using this to mean "definitely internal stuff" but yeah it's not documented for Tahoe-LAFS :/ )